packages/topkit-analytics-charts/src/CategoriesChart/CategoriesChart.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useMemo } from 'react'
+import { BarChartProps, BarChart } from '@toptal/picasso-charts'
+
+import { Bar, Labels, Tooltips } from './types'
+import { formatData, getColor } from './utils'
+import CategoriesChartTooltip from '../CategoriesChartTooltip'
+
+export type Props = Pick<BarChartProps<string>, 'width' | 'height'> & {
+  data: Bar[]
+  labels: Labels
+  tooltips: Tooltips
+}
+
+export const CategoriesChart = ({
+  data,
+  labels,
+  tooltips,
+  ...restProps
+}: Props) => {
+  const chartData = useMemo(() => formatData(data, labels), [data, labels])
+
+  const getBarLabelColor = ({
+    index,
+    dataKey
+  }: {
+    index?: number
+    dataKey: string
+  }) =>
+    index !== undefined ? getColor({ dataKey, entry: chartData[index] }) : ''
+
+  return (
+    <BarChart
+      labelKey='label'
+      data={chartData}
+      tooltip
+      getBarLabelColor={getBarLabelColor}
+      customTooltip={
+        // @ts-expect-error: There is some magic in recharts that adding the props for us
```

Comment:
Can you explain this part in detail, please?

Reply:
There are 2 other props Recharts passes to this tooltip component under the hood: `active` and `payload`. We need to assume they are there without explicitly passing them. Please see https://recharts.org/en-US/api/Tooltip#active : `..., usually calculated internally.`

Reply:
Let's replace the comment then

```suggestion
        // @ts-expect-error: Recharts passes 2 props under the hood: active and payload. We need to assume they are there without explicitly passing them.
```